### ISSUE TITLE
Enable updates of README.md files

### ISSUE THREAD

**Issue by 2ndkauboy** [10up/action-wordpress-plugin-asset-update#1]
The assets update action currently only handles changes on a `readme.txt` file. Nowadays many plugin developers use a `README.md` file instead, escpecially when hosting the plugin code on a git hosting platform like GitHub.

I would suggest to add a environment variable to enable users to set the file name of the readme file to a different name, so this file is checked for changes.

**Comment by 2ndkauboy** [comment]
Is there anything that's missing with this issue? I would really love to see it merged :)

**Comment by helen** [comment]
Fixed in #3

**Comment by tedmasterweb** [comment]
I know this is closed but thought I should chime in given a recent experience. I just published a [new plugin](https://wordpress.org/plugins/identify-headings/). I only had a `.md` file, no `.txt`. The ingest did not seem to work. The readme is not being displayed and none of the tags are recognized. This could be an issue with the first commit, but, hard to say.

That said, I have [another plugin](https://wordpress.org/plugins/ratify/) that also only has a `.md` file and it seems to be working fine, but I'm pretty sure that the original version was `.txt` and I changed it when I moved the plugin to github.

HTH
